docs/ossec_alerts.rst

```diff
@@ -90,39 +90,44 @@ install\_files/ansible-base and then specify the filename in the
 If you don't have your GPG key ready, you can run GnuPG on the command
 line in order to find, import and export your public key. It's best to
 copy the key from a trusted and verified source, but you can also
-request it from keyservers using the known longid. Looking it up by
-email address could cause you to obtain a wrong, malicious or expired
-key. Just replace the key ID in the following example with yours:
+request it from keyservers using the known fingerprint. Looking it up by
+email address or a shorter key ID format could cause you to obtain a
+wrong, malicious or expired key. Just replace the word ``<fingerprint>``
+in the following example with your own:
 
-Download your key and import it into the local keyring:
+Download your key and import it into the local keyring: ::
 
-::
+    gpg --keyserver <fingerprint> hkp://qdigse2yzvuglcix.onion --recv-key C5986B4F1257FFA86632CBA746181433FBB75451
 
-    gpg --recv-keys 0xFD720AD9EBA34B1C
+As in the setup for the *Admin Workstation*, we encourage you to first
+write out the fingerprint in groups of four, double-check you have
+entered it correctly, remove the spaces, and then press "enter" to
+actually fetch the key (and implicitly verify the fingerprint of the key
+received matches the one requested).
```

Comment:
Tested it on Tails and it works. It seems like this is going nowhere on the GPGv1 branch, which Tails is not moving off of any time soon. I think it would be fine to use this command for the time being if we file an issue upstream asking them to add it to GPGv2. Since even Jessie defaults to GPGv1, it's going to be at least 2 years before we might have to consider this again.